storage/src/tracking_copy/mod.rs

```diff
@@ -485,9 +511,13 @@ where
 
     /// Get record by key.
     pub fn get(&mut self, key: &Key) -> Result<Option<StoredValue>, TrackingCopyError> {
-        if let Some(value) = self.cache.get(key) {
+        let get = self.cache.get(key);
```

Comment:
nit
```suggestion
        let cache_entry = self.cache.get(key);
```

Also, `match cache_entry` here could improve readability